languages/rust/exercises/concept/vec-macro/tests/vec_macro.rs

```diff
@@ -0,0 +1,21 @@
+#[cfg(test)]
+mod tests {
+    use vec_macro::*;
+    #[test]
+    fn test_empty() {
+        assert_eq!(create_empty(), Vec::new());
+    }
+    #[test]
+    fn test_buffer() {
+        let mut zeroized = Vec::new();
+        zeroized.extend_from_slice(&[0, 0, 0]);
+        assert_eq!(create_buffer(3), zeroized);
+    }
+    #[test]
+    fn test_fibonacci() {
+        let mut first_five = Vec::new();
+        first_five.extend_from_slice(&[1, 1, 2, 3, 5]);
+
+        assert_eq!(fibonacci(), first_five);
+    }
```

Comment:
```suggestion
    fn test_fibonacci() {
        let fibb = fibonacci();
        assert_eq!(fibb.len(), 5);
        for window in fibb.windows(3) {
            assert_eq!(window[0] + window[1], window[2]);
        }
    }
```

Reduces chance of copy/paste from the answer.